myia/api.py

```diff
@@ -50,10 +51,14 @@ def default_object_map() -> Dict[Any, ANFNode]:
         operator.ge: Constant(P.ge),
         operator.not_: Constant(P.not_)
     }
+    for prim, impl in implementations.items():
```

Comment:
I would prefer not exposing this right now. I think the decision as to whether we want shadow NumPy's API, or have our own API, and if so, what that API should look like, it still an open question. And for now I think all the primops are covered by Python operators, right?

Reply:
`getitem/setitem` aren't at the moment, but it's true they are in the `operator` package. I wouldn't really call this "exposing" them, though -- this only makes it so that if we import `myia.py_implementations.add` and use it, it'll resolve to the primitive instead of naively building a graph for it. There is an internal use case for this, which is that if we define some primitive's gradient as a Myia graph, and it needs a primitive that isn't covered by Python's standard library, we can simply get it from there. So while I agree it probably shouldn't be a user-facing API, it's still internally useful.